pkg/daemon/server/daemon_server.go

```diff
@@ -86,7 +86,7 @@ func (ds *daemonServer) Run(ctx context.Context) error {
 	default:
 		return fmt.Errorf("unsupported isbsvc buffer type %q", ds.isbSvcType)
 	}
-	wmFetchers, err := service.GetEdgeWatermarkFetchers(ctx, ds.pipeline, isbSvcClient)
+	wmFetchers, err := service.GetUXEdgeWatermarkFetchers(ctx, ds.pipeline, isbSvcClient)
```

Comment:
Can we get `UX` out of the function name? It limits the usage of the function only for UX purpose.

Reply:
What I meant was, use a different word from `UX`, otherwise it will be tricky if somehow some other services need to use it in the future.

Reply:
should we be exposing it for other use-cases? as of now only UX uses it and ideally nobody should be using it.

Reply:
For example, we write a CLI someday, it displays the status of a pipeline, which includes the watermark of each edge, then it most probably uses the same interface/function.

Reply:
we can rename it then? the only concern I have is that people might misuse it without knowing the side effects.

Reply:
Maybe we can rename later if we are to add new use cases?... Can't think of a proper name for UXFetcher and related methods/functions............. `Fetcher` and `SimpleFetcher` ?..